scalastyle_config.xml

```diff
@@ -0,0 +1,266 @@
+<!--
+Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
```

Comment:
what is the origin of this file?

Reply:
It's a subset of [Spark's style](https://github.com/apache/spark/blob/master/scalastyle-config.xml)